include/wx/ribbon/buttonbar.h

```diff
@@ -202,6 +203,7 @@ class WXDLLIMPEXP_RIBBON wxRibbonButtonBar : public wxRibbonControl
     wxRibbonButtonBarButtonInstance* m_active_button;
 
     wxPoint m_layout_offset;
+    // Sizes are the logical size
```

Comment:
I don't think we should this comment, _all_ sizes are in logical units in wx API, unless explicitly mentioned otherwise, so having it just here, and not in a million of other places just like this, is inconsistent and even somewhat misleading, as it implies this one is somehow different.
```suggestion
```